config/cfme/manifest_5_11.json

```diff
@@ -24,58 +24,64 @@
       "vms": {
         "id": null,
         "name": null,
+        "description": null,
         "type": null,
-        "guid": null,
+        "ems_ref": null,
         "uid_ems": null,
         "archived": null,
-        "cpu_cores_per_socket": null,
+        "orphaned": null,
+        "retired": null,
         "cpu_total_cores": null,
+        "cpu_cores_per_socket": null,
         "disks_aligned": null,
-        "ems_ref": null,
         "has_rdm_disk": null,
-        "host": {
-          "ems_ref": null
-        },
-        "linked_clone": null,
-        "orphaned": null,
         "power_state": null,
+        "ipaddresses": null,
+        "hostnames": null,
```

Comment:
I think these two are redundant if we're getting ipaddress and hostname from networks, these are essentially just `vm.hardware.networks.ipaddress` and `vm.hardware.networks.hostname`